lib/routes/lnurl/payment/lnurl_payment_page.dart

```diff
@@ -127,17 +128,21 @@ class LNURLPaymentPageState extends State<LNURLPaymentPage> {
               ),
               if (!fixedAmount) ...[
                 Padding(
-                  padding: const EdgeInsets.only(
-                    top: 8,
-                  ),
+                  padding: const EdgeInsets.only(top: 8),
                   child: RichText(
                     text: TextSpan(
                       style: theme.FieldTextStyle.labelStyle,
                       children: <TextSpan>[
                         TextSpan(
                           text: texts.lnurl_fetch_invoice_min(
                             currencyState.bitcoinCurrency.format(
-                              (widget.data.minSendable.toInt() ~/ 1000),
+                              max(
+                                liquidMinimumPaymentAmountSat,
+                                max(
+                                  context.read<LnUrlCubit>().state.limits?.send.minSat.toInt() ?? 0,
+                                  widget.data.minSendable.toInt() ~/ 1000,
```

Comment:
What is the difference between the two values here?

Reply:
> What is the difference between the two values here?

@roeierez 
- [data.minSendable](https://github.com/lnurl/luds/blob/luds/06.md#pay-to-static-qrnfclink) is the minimum amount the LN service is willing to receive.

- `Limits` on `LightningPaymentLimitsResponse` is either normal or reverse submarine swap limits returned from Boltz service.

Both values can be below 1000 sats.